example_test.go

```diff
@@ -124,6 +124,9 @@ type MyCustomClaims struct {
 	jwt.RegisteredClaims
 }
 
+// Ensure we implement [jwt.ClaimsValidator] at compile time so we know our custom Validate method is used.
+var _ jwt.ClaimsValidator = MyCustomClaims{}
```

Comment:
nit, can we change this to `(*MyCustomClaims)(nil)` ?

Reply:
```suggestion
var _ jwt.ClaimsValidator = (*MyCustomClaims)(nil)
```